datastore/index.js

```diff
@@ -1,29 +1,45 @@
+const fs = require('fs');
+
+let _data = { 'subscriptions': [] };
+const _datastore_path = 'db.json';
+
 module.exports = {
-  async add({ channelId, githubProjectUrl }) {
-    // TODO save to database
-    console.log(`Subscibe ${channelId} to ${githubProjectUrl}`);
+  async connectToDatastore() {
+    fs.readFile(_datastore_path, (err, data) => {
+      if (err) {
+        if (err.code === 'ENOENT') {
+          // Create local datastore
+          module.exports.commit();
+        }
+        else {
+          throw err;
+        }
+      }
+      else {
+        _data = JSON.parse(data);
+      }
+    });
   },
-  async remove({ channelId, githubProjectUrl }) {
-    // TODO remove from database
-    console.log(`Unsubscribe ${channelId} to ${githubProjectUrl}`);
+  add({ channelId, githubProjectUrl, commit = true }) {
+    _data.subscriptions.push({ channelId: channelId, githubProjectUrl: githubProjectUrl });
+    if (commit) {
+      module.exports.commit();
+    }
   },
-  async queryByGitHubProjectUrl(githubProjectUrl) {
-    // TODO
-    console.log(`Find by githubProjectUrl ${githubProjectUrl}`);
-    const results = [];
-    if (process.env.DEBUG_CHANNEL) {
-      results.push(
```

Comment:
We do still need to add DEBUG_CHANNEL somewhere. I see you removed it here do you have a better place where you feel is more appropriate?

Reply:
Do we want to send properly styled messages (```MessageEmbed```)? If so, it might be better to call the debug hook from the webhook module.
If we do not want styled messages but simple JSON strings with method names to be sent on ```DEBUG_CHANNEL```, we can call the debug hook here.